dir.props

```diff
@@ -548,7 +552,7 @@
     <IntermediateOutputRootPath Condition="'$(IntermediateOutputRootPath)' == ''">$(BaseIntermediateOutputPath)$(OSPlatformConfig)/</IntermediateOutputRootPath>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateOutputRootPath)$(MSBuildProjectName)/$(TargetOutputRelPath)</IntermediateOutputPath>
 
-    <TestPath Condition="'$(TestPath)'==''">$(TestWorkingDir)$(OSPlatformConfig)/$(MSBuildProjectName)/$(TargetOutputRelPath)</TestPath>
+    <TestPath Condition="'$(TestPath)'==''">$(TestWorkingDir)$(ProjectRunConfig)/$(MSBuildProjectName)/$(TargetOutputRelPath)</TestPath>
```

Comment:
For now I would just update TargetOutputRelPath (or another property) to also include TestTFM. Once we move these to buildtools we can start having properties trying to combine the properties in logical groups that people can depend on.